pkg/crud/service_test.go

```diff
@@ -17,6 +17,15 @@ const (
 	dataTags     = "fixtures/tags.json"
 )
 
+func init() {
+	mm := GetMongoManager()
```

Comment:
Are you not having any problem with this init versus the init in service.go (crud package) ?